pkg/apiserver/utils/iter.go

```diff
@@ -1,4 +1,17 @@
-// Copyright 2017 EasyStack, Inc.
+// Copyright 2023 Linkall Inc.
```

Comment:
if copy third-part code, don't change license header

Reply:
This is only the underlying part of the encapsulation logic, not involving third-party code

Reply:
why this file copyright is EasyStack before this PR?